src/sandbox/alternative-threading/CountDownLatch2.cs

```diff
@@ -101,11 +101,11 @@ public bool await(TimeSpan timeout)
 		/// Waits the specified amount of time for the latch
 		/// returning true if the latch was acquired
 		/// <summary>
-        public bool await(int millis)
+        public bool Await(int millis)
```

Comment:
This was causing some issues because of the reserved keyword `await` (in the AsyncAwait TPL)